### DISCUSSION TITLE
Winforms MessageBox.Show equivalent

### DISCUSSION THREAD

**Discussion by Balkoth** [microsoft/WindowsAppSDK#1965] [Q&A] [answered]
Sorry if this is a stupid question, but how to show a messagebox the winui3 way?

**Comment by castorix** [comment, marked as answer]
For example with [ContentDialog ](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.contentdialog?view=winui-3.0)
